### PR TITLE
Use f-strings

### LINKED ISSUE #54: switch to f-strings
We now require Python 3.6 for pybv and we can thus switch to using fstrings like: `print(f"this nice value: {value}")` instead of `print("this nice value: {}".format(value))`

@hoechenberger and I think that that'll make the codebase more readable.

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #54. Please double-check if all changes are OK (there were some tricky ones).

### CODE REVIEW

File: pybv/io.py [hunk 1]
```diff
@@ -108,21 +108,19 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
...
+        raise ValueError("Channel names must be unique, found duplicate name.")
 
     if not isinstance(sfreq, (int, float)):
         raise ValueError("sfreq must be one of (float | int)")
     sfreq = float(sfreq)
 
     resolution = np.atleast_1d(resolution)
     if not np.issubdtype(resolution.dtype, np.number):
-        raise ValueError("Resolution should be numeric, is {}".format(resolution.dtype))  # noqa: E501
+        raise ValueError("Resolution should be numeric, is {resolution.dtype}")
```
Comment (hoechenberger) [inline pybv/io.py:123]:
```suggestion
        raise ValueError(f"Resolution should be numeric, is {resolution.dtype}")
```

**Review by hoechenberger — APPROVED** [review]

File: pybv/io.py [hunk 2]
```diff
@@ -108,21 +108,19 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
...
+        raise ValueError("Channel names must be unique, found duplicate name.")
 
     if not isinstance(sfreq, (int, float)):
         raise ValueError("sfreq must be one of (float | int)")
     sfreq = float(sfreq)
 
     resolution = np.atleast_1d(resolution)
     if not np.issubdtype(resolution.dtype, np.number):
-        raise ValueError("Resolution should be numeric, is {}".format(resolution.dtype))  # noqa: E501
+        raise ValueError(f"Resolution should be numeric, is {resolution.dtype}")
```
Comment (sappelhoff) [inline pybv/io.py:123]:
```suggestion
        raise ValueError(f"Resolution should be numeric, is {resolution.dtype}")  # noqa:E501
```

**Review by sappelhoff — APPROVED** [review]
LGTM, thanks.

I assume you removed the "raw" identifier in front of strs (`r"str"`) because it was not really needed?

**Comment by cbrnr** [comment]
> I assume you removed the "raw" identifier in front of strs (`r"str"`) because it was not really needed?

Yes.

**Pull request merged**